<ui version="4.0" >
 <class>UserProfileListManagerWidget</class>
 <widget class="QWidget" name="UserProfileListManagerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PyWengoPhone - My UserProfileList</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Current</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Id</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="2" >
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QPushButton" name="deleteButton" >
       <property name="text" >
        <string>Delete</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="modifyButton" >
       <property name="text" >
        <string>Modify</string>
       </property>
       <property name="icon" >
        <iconset/>
       </property>
      </widget>
     </item>
     <item row="0" column="5" >
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="loadButton" >
       <property name="text" >
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>modifyButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
